worker.config.json

```diff
@@ -5,5 +5,10 @@
         "defaultExecutablePath":"node",
         "defaultWorkerPath":"dist/src/nodejsWorker.js",
         "workerIndexing": "true"
+    },
+    "processOptions": {
+        "initializationTimeout": "00:05:00",
```

Comment:
Since these timeouts are "new" and pretty much unique to the new model, we have an opportunity to do something shorter than what we did in the old model. 5 minutes feels egregious to me, and I kind of want to put more pressure on people to use `WEBSITE_RUN_FROM_PACKAGE`. People don't realize how much happier they will be if they just add the setting

To me, 1 minute feels like a reasonable timeout. But I normally double what I think is reasonable to account for unexpected edges cases so I guess I'd vote for 2 minutes.

IMO we can always increase this value in the future, but I don't think we can decrease it.

Reply:
I don't have strong feelings here so I'm happy to defer to your experience and reduce to 2 minutes

Reply:
Done